lib/src/metta/runner/stdlib.metta

```diff
@@ -55,46 +62,126 @@
+;    (@param "String containing return value description")))
+;  (@return "Return value description"))
 (: @return (-> String DocReturnInformal))
+(: @return (-> DocType DocDescription DocReturn))
 
+(@doc @doc-formal
+  (@desc "Used for documentation purposes. get-doc returns documentation starting with @doc-formal symbol. @doc-formal contains 6 or 4 parameters depending on the entity being described (functions being described using 6 parameters, atoms - 4 parameters)")
+  (@params (
+    (@param "DocItem. Function/Atom name for which documentation is to be displayed. Format (@item name)")
+    (@param "DocKindAtom/DocKindFunction. Contains (@kind function) or (@kind atom) depends on entity which documentation is displayed")
+    (@param "DocType. Contains type notation of function/atom")
+    (@param "DocDescription. Function/atom description")
+    (@param "DocParameters (functions only). Description of function parameters")
+    (@param "DocReturn (functions only). Description of function's return value")))
+  (@return "Expression containing full documentation on function"))
 (: @doc-formal (-> DocItem DocKindFunction DocType DocDescription DocParameters DocReturn DocFormal))
 (: @doc-formal (-> DocItem DocKindAtom DocType DocDescription DocFormal))
+
+(@doc @item
+  (@desc "Used for documentation purposes. Converts atom/function's name to DocItem")
+  (@params (
+    (@param "Atom/Function name to be documented")))
+  (@return "(@item Atom) entity"))
 (: @item (-> Atom DocItem))
+
+; get-doc not showing docs for following two functions (@kind function) and (@kind atom) for some reason.
+(@doc (@kind function)
+  (@desc "Used for documentation purposes. Shows type of entity to be documented. (@kind function) in this case")
+  (@return "(@kind function) entity"))
 (: (@kind function) DocKindFunction)
+
+(@doc (@kind atom)
+  (@desc "Used for documentation purposes. Shows type of entity to be documented. (@kind atom) in this case")
+  (@return "(@kind atom) entity"))
 (: (@kind atom) DocKindAtom)
+
+(@doc @type
+  (@desc "Used for documentation purposes. Converts atom/function's type to DocType")
+  (@params (
+    (@param "Atom/Function type to be documented")))
+  (@return "(@type Type) entity"))
 (: @type (-> Type DocType))
-(: @params (-> Expression DocParameters))
-(: @param (-> DocType DocDescription DocParameter))
-(: @return (-> DocType DocDescription DocReturn))
 
+(@doc @params
+  (@desc "Used for function documentation purposes. Contains several @param entities with description of each @param")
+  (@params (
+    (@param "Several (@param ...) entities")))
+  (@return "DocParameters containing description of all parameters of function in form of (@params ((@param ...) (@param ...) ...))"))
+(: @params (-> Expression DocParameters))
 
+(@doc get-doc
+  (@desc "Returns documentation for the given Atom/Function. It checks if given variable is an expression and evaluates get-doc-atom on the input value or get-doc-single-atom otherwise")
```

Comment:
One can see the second part from the code of the function. Here the goal is to explain what function should be used for.
```suggestion
  (@desc "Returns documentation for the given Atom/Function")
```

Reply:
Done